src/compute.client/PythonClient.cs

```diff
@@ -430,7 +438,8 @@ protected override string ToComputeClient(ClassBuilder cb)
                         }
                         if (baseClass.ClassName == "Point3d" ||
                             baseClass.ClassName == "Vector3d" ||
-                            baseClass.ClassName == "Line")
+                            baseClass.ClassName == "Line" ||
+                            baseClass.ClassName == "BoundingBox")
```

Comment:
This is never called because `ClassBuilder.Get("BoundingBox")` returns `null` – opennurbs_bounding_box.cs doesn't contain any `#if RHINO_SDK...#endif` spans, so it isn't added to `ClassBuilder.AllClasses`.